collector/__init__.py

```diff
@@ -74,7 +74,7 @@ def ot_timing_windows(starts: Iterable[int],
     return timing_windows
 
 
-def select_obsclass(classes):
+def select_obsclass(classes: List[str])-> str:
```

Comment:
We really should turn the obsclasses into a proper `Enum` since there are a limited number of them.
@bryanmiller Could you comment on what possible values there are for obsclasses?

Reply:
I'm a little confused, then. Why are we passing the `classes` as a `List[str]` and not a `List[ObservationClass]`?

Reply:
Never mind: I see further ahead what is going on.